### DISCUSSION TITLE
Switch to AC - cannot find auth and characters databases for backup

### DISCUSSION THREAD

**Discussion by Durddill** [azerothcore/azerothcore-wotlk#12918] [General]
Hi there,

I am new to compiling and starting a small server for a few friends and myself, so first of all thanks for all the work you did.

I first saw the trinitycore project and started with that but now more interested by the AC project I am trying to switch, following this guide https://www.azerothcore.org/wiki/trinitycore-to-azerothcore-characters-migration. Step 1 is done, but I am confused regarding step 2, when I list databases with mysql, i see the "world" one but not the "auth" nore the "characters" one, any clue?

Thanks again.

**Comment by LukasVolgger** [comment]
Hi!  The databases have to be there, otherwise you wouldn't have been able to play already.  The AC and TC installation guide describes exactly how the DB must be installed.  Just connect with a DB tool like Heidi SQL and there you have the databases.

**Reply by Durddill** [reply]
Hi Lukas, I agree with you, db have to be there, that is why I am confused.

Here attached is the list of the db within Heidisql:
![db_list](https://user-images.githubusercontent.com/8124102/187787686-8b18ce64-6bb1-466f-9250-1a9e2bd50d14.PNG)

**Reply by LukasVolgger** [reply]
Have you another version of MySQL installed for TC? Check the authserver.conf and look what port and user credentials are in there

**Reply by Durddill** [reply]
Hi, only one version of MySQL here.
I tried the credentials there but authentication fails. I was using other credentials with trinitycore (I should have said it was a repack downloaded from ac-web, I did not compile a trinitycore build on my own) and those previous credentials do not work anymore.
I have no idea what I did that caused such a mess, I was level 18 with my new main so that's ok I think.
Anyway thank you for your time.

**Reply by LukasVolgger** [reply]
In that case you should be able to start the MySQL in your Repack with mysql.bat or something and do the same way. Repacks are using a portable MySQL server and are technically only pre-compiled binaries from a repo with modifications.

But if you start witz a fresh DB you will have less worries to play because i'm not sure if the migration tool works properly.

Have fun with AC!

**Reply by Durddill** [reply]
Thanks Lukas.
